airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/objects/serde/ConnectionSpecificationSerializer.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol.objects.serde;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import io.airbyte.commons.protocol.objects.ConnectorSpecification;
+import java.io.IOException;
+
+public class ConnectionSpecificationSerializer extends StdSerializer<ConnectorSpecification> {
+
+  public ConnectionSpecificationSerializer() {
+    this(null);
+  }
+
+  protected ConnectionSpecificationSerializer(Class<ConnectorSpecification> t) {
+    super(t);
+  }
+
+  @Override
+  public void serialize(ConnectorSpecification value, JsonGenerator gen, SerializerProvider provider) throws IOException {
+    gen.writeStartObject();
+    gen.writeStringField("object", value.toJson());
```

Comment:
which components would receive these `{object: {spec...}}` messages? I.e. do connectors need to produce this format? Or are connectors still expected to produce protocol messages, and then the worker would translate them into this format? (or something else?)

Reply:
I think those should remain within the platform, those objects should be mostly used for temporal payloads which is generated by the clients we use and the activity. It currently leaks to the connector's tests because some tests rely on the workers.
Connectors having to generate those should be a red flag.